test/ControlPanel.vala

```diff
@@ -147,6 +158,7 @@ namespace BrickManager {
                 file_browser_controller = new FakeFileBrowserController (builder);
                 device_browser_controller = new FakeDeviceBrowserController (builder);
                 network_controller = new FakeNetworkController (builder);
+                audio_controller = new FakeAudioController(builder);
```

Comment:
style: space between function name and `(`. This applies to all new files as well.